CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Development
+# 5.0.3
 
 - Change HMRC and DEFRA text colours for improved contrast
```

Comment:
The other entries in the changelog include a link to the PR that introduced the changes.